.github/workflows/ci.yml

```diff
@@ -168,7 +168,7 @@ jobs:
           name: snapshots
           path: packages/react-composites/test-results
       - name: Comment on PR
-        if: ${{ github.event_name == 'pull_request' && always() && steps.visualregressiontests.outcome == 'failure' && !contains( github.event.pull_request.labels.*.name, 'ui change') }}
+        if: ${{ github.event_name == 'pull_request' && always() && steps.visualregressiontests.outcome == 'failure' && !contains( github.event.pull_request.labels.*.name, 'update_snapshots') }}
```

Comment:
To reduce re-education on updating snapshots can we keep the old label name? (`ui change`)

Reply:
I intentionally chose a new label for two reasons:

* Allows me to introduce this new workflow without affecting the old one - gentle transition
* I want to reeducate folks. If I keep the same label, folks might not notice that anything changed, and may be confused when the label auto-disappears from their PRs.